lib/rubocop/factory_bot/version.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module RuboCop
-  module RSpec
-    # Version information for the RSpec RuboCop plugin.
+  module FactoryBot
+    # Version information for the FactoryBot RuboCop plugin.
     module Version
       STRING = '2.18.1'
```

Comment:
Maybe start with 2.19.0, as it would correspond to the version of `rubocop-rspec` where it will be extracted?

Reply:
Or 2.20.0 if we decide to release what we currently have unreleased in 2.19.0. WDYT @bquorning ?